Resources/Textures/_DV/Objects/Tools/syndielighter.rsi/meta.json

```diff
@@ -0,0 +1,45 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Sprites Created by JustAnOrange",
+    "size": {
+      "x": 32,
+      "y": 32
+    },
+    "states": [
+      {
+        "name": "SyndieLighterFire",
```

Comment:
rename the states to not have SyndieLighter prefix and not be PascalCase
- fire
- icon
- open
- top
- inhand-left-lit
- inhand-right-lit